airflow/providers/amazon/aws/triggers/sagemaker.py

```diff
@@ -41,7 +42,7 @@ def __init__(
         job_name: str,
         job_type: str,
         poke_interval: int = 30,
-        max_attempts: int | None = None,
+        max_attempts: int = 480,
```

Comment:
passing None was not working before, so this is not a breaking change since it was already broken.